.github/semantic.yml

```diff
@@ -0,0 +1,14 @@
+# Always validate the PR title, and ignore the commits
+titleOnly: true
+# Always validate the PR title AND all the commits
+titleAndCommits: false
+# You can define a list of valid scopes
+scopes:
+  - cli
+  - server
+  - utils
+  - viewer
+  - recipes
+# Allows use of Merge commits (eg on github: "Merge branch 'master' into feature/ride-unicorns")
+# this is only relevant when using commitsOnly: true (or titleAndCommits: true)
```

Comment:
does this mean we don't need this flag at all?

Reply:
Yep, we don't need it.

- [x] Fixed in 80d2a34